api/src/main/resources/workbench-api.yaml

```diff
@@ -5161,12 +5161,16 @@ definitions:
   RasLinkRequestBody:
     required:
     - authCode
+    - redirectUrl
     type: object
     properties:
       authCode:
         type: string
         description: 'the standard OAuth authorization code from RAS that we can use to exchange
         access token.'
+      redirectUrl:
+        type: string
+        description: 'The redirect url that was used when generating the authorization code.'
```

Comment:
I'm not following this part - where are we getting this, how are we using it, and what is an example value in practice?

Reply:
That is a tricky one, and I am not sure if I handled it correctly. 
(1) That is not used(the redirect URI happens in UI), but RAS requires it even for the following code exchange call (with &refirect_uri=), and that is the common pattern for OAuth. So I have to set it correctly to make RAS happy. 
(2) I could put it in config file, but I am not sure if that is better. It is relative easier for UI to get it, and just set it in the request. (in UI, it can grab the current url then append the static /ras-callback). So I just follow Bond's pattern https://github.com/DataBiosphere/bond/blob/develop/swagger/api-docs.yaml#L169. Terra UI grab the redirect uri, then Bond just consume it.

Reply:
What is an example value, for a request in the test env?